pontoon/administration/views.py

```diff
@@ -365,8 +367,10 @@ def _create_or_update_translated_resources(
         )
 
     if resource is None:
+        is_tutorial = project.slug == 'tutorial'
+        resource_path = 'playground' if is_tutorial else 'database'
         resource, _ = Resource.objects.get_or_create(
-            path='database',
+            path=resource_path,
```

Comment:
A DB project can only have 1 resource. It would be much more elegant to rely on that instead of hardcoding resource paths (twice).